generate-spec

```diff
@@ -338,6 +339,8 @@ foreach ($parsedRoutes as $key => $value) {
 		if ($methodFunction == null || $isIgnored) {
 			if (!$isIgnored) {
 				Logger::warning($routeName, "Route ignored");
+			} else {
+				Logger::info($routeName, "Route ignored because of IgnoreOpenAPI attribute");
 			}
 			continue;
 		}
```

Comment:
```suggestion
		if ($methodFunction == null) {
			Logger::warning($routeName, "Route ignored");
			continue;
		}
		if ($isIgnored) {
			Logger::info($routeName, "Route ignored because of IgnoreOpenAPI attribute");
			continue;
		}
```
This is a bit cleaner and removes the nested logic

Reply:
I wonder if `$methodFunction == null` should be an error? It sounds kind of broken?

Reply:
Hm true, I'm not sure why I did it this way. Maybe there was an app that had a broken route, but then it should be fixed in the routes.php instead. Can you make `Logger::panic($routeName, "Missing controller method")` instead?